src/Modules/Finance/Header/Transaction/Transaction.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import styles from '../Form.module.css';
+
+interface ITransactionProps {
+	value: string,
+	clicked: (event: React.MouseEvent<HTMLButtonElement>) => void,
+	transactor: string,
+	amountChanged: (event: React.ChangeEvent<HTMLInputElement>) => void,
+	nameChanged: (event: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+export const Transaction: React.FC<ITransactionProps> = props => {

```

Comment:
Wrap all the functional components in `React.memo` :)
https://reactjs.org/docs/react-api.html#reactmemo

Reply:
If there were class components, I could use React.PureComponent instead of React.memo, right?

Reply:
Done